setup.cfg

```diff
@@ -96,6 +96,7 @@ development =
     types-appdirs >= 1.4.3
     types-invoke >= 1.7.3
     types-protobuf >= 4.21.0
+    wasmtime == 1.0.0
```

Comment:
It seems that the current version of `wasmtime` is `20.0.0`. Is `1.0.0` really what's needed here?

Reply:
Somehow I always get 1.0.0 when searching on Google 🤔 Updated.